### PR TITLE
adding multiple uploads to ps example script

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: powershell/upload/UploadFile.ps1 [hunk 1]
```diff
@@ -80,17 +80,24 @@ catch {
...
+    try {
+        $createdFilePath = $fileApiService.CreateFileToUpload($_.FullName) 
+        $fileApiService.UploadFile($createdFilePath, $(Split-Path -Path $_.FullName -Leaf))
+        $archivedFile =  [Helper]::ArchiveFile($config.Upload.ArchivePath, $_.FullName)
+        if(-not [string]::IsNullOrEmpty($archivedFile)){
+            Write-Host "File archived ($($archivedFile))."
+        }
+    }
+    catch {
+        [Helper]::EndProgramWithError($_, "Failure uploading the file.")
```
Comment (danielcanamares) [inline powershell/upload/UploadFile.ps1]: Is it possible that the error message contains the name of the file?

Comment (danielcanamares) [inline powershell/upload/UploadFile.ps1]: Is it possible that the error message contains the name of the file?

File: powershell/upload/UploadFile.ps1 [hunk 2]
```diff
@@ -118,15 +125,19 @@ class ConfigurationManager {
...
     
         $missingConfiguration = @()
         if ([string]::IsNullOrEmpty($credentialsPath)) { $missingConfiguration += "Credentials.Path" }
         if ([string]::IsNullOrEmpty($fileApiBaseUrl)) { $missingConfiguration += "Services.FileApiBaseUrl" }
         if ([string]::IsNullOrEmpty($authenticationTokenApiBaseUrl)) { $missingConfiguration += "Services.AuthenticationTokenApiBaseUrl" }
         if ([string]::IsNullOrEmpty($tenantId)) { $missingConfiguration += "Upload.TenantId" }
         if ([string]::IsNullOrEmpty($businessTypeId)) { $missingConfiguration += "Upload.BusinessTypeId" }
-        if ([string]::IsNullOrEmpty($contentFilePath)) { $missingConfiguration += "Upload.Path" }
+        if ([string]::IsNullOrEmpty($contentDirectoryPath)) { $missingConfiguration += "Upload.Path" }
+        if ([string]::IsNullOrEmpty($contentFilter)) { $missingConfiguration += "Upload.Filter" }
```
Comment (danielcanamares) [inline powershell/upload/UploadFile.ps1:148]: Validation for ArchivePath is missing.

Comment (danielcanamares) [inline powershell/upload/UploadFile.ps1:148]: Validation for ArchivePath is missing.

File: powershell/upload/UploadFile.ps1 (hunk 1 above)
Comment (ghost) [reply powershell/upload/UploadFile.ps1]: Added filename to errormessage

Comment (ghost) [reply powershell/upload/UploadFile.ps1]: Added filename to errormessage

File: powershell/upload/UploadFile.ps1 (hunk 2 above)
Comment (ghost) [reply powershell/upload/UploadFile.ps1:148]: Added Validation

Comment (ghost) [reply powershell/upload/UploadFile.ps1:148]: Added Validation

**Review by danielcanamares — APPROVED** [review]

**Pull request merged**